hoot-core-test/src/test/cpp/hoot/core/ops/UnconnectedWaySnapperTest.cpp

```diff
@@ -98,9 +100,9 @@ class UnconnectedWaySnapperTest : public HootTestFixture
     MapProjector::projectToWgs84(map);
     OsmXmlWriter().write(map, outputPath + testName +  + "Out.osm");
 
-    CPPUNIT_ASSERT_EQUAL(uut.getNumAffected(), 41L);
-    CPPUNIT_ASSERT_EQUAL(uut.getNumSnappedToWayNodes(), 5L);
-    CPPUNIT_ASSERT_EQUAL(uut.getNumSnappedToWays(), 36L);
+    CPPUNIT_ASSERT_EQUAL(43L, uut.getNumAffected());
+    CPPUNIT_ASSERT_EQUAL(5L, uut.getNumSnappedToWayNodes());
+    CPPUNIT_ASSERT_EQUAL(38L, uut.getNumSnappedToWays());
```

Comment:
First off `CPPUNIT_ASSERT_EQUAL` takes expected then output so I reversed the variables.  Also my fixes made the number of snapped ways increase.